bindings/protocols/coap/context.jsonld

```diff
@@ -0,0 +1,38 @@
+{
+    "@context": {
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "cov": "https://www.w3.org/2019/wot/coap#",
+        "method": {
+            "@id": "cov:method",
+            "@type": "xsd:string"
+        },
+        "blockwise": {
+            "@id": "cov:blockwise",
+            "@type": "@vocab"
+        },
+        "qblockwise": {
+            "@id": "cov:qblockwise",
+            "@type": "@vocab"
+        },
```

Comment:
Should this be referring to the `BlockWiseTransferParameters` class somehow?

Reply:
I've tried a few variants in the JSON-LD playground but couldn't get it to emit a `"@type": "cov:BlockWiseTransferParameters"` if there is no `@type` in the data. Setting the `@type` to a class in an @context doesn't seem to be doing anything.

<details>
<summary>Details (click to expand)</summary>

```turtle
{
  "@context": {
    "cov": "https://www.w3.org/2019/wot/coap#",
    "hctl": "https://www.w3.org/2019/wot/hypermedia#",
    "xsd": "http://www.w3.org/2001/XMLSchema#",
    "Form": {
      "@id": "hctl:Form",
      "@context": {
        "blockwise": {
          "@id": "cov:blockwise"
        }
      }
    },
    "BlockWiseTransferParameters": {
      "@id": "cov:BlockWiseTransferParameters",
      "@context": {
        "blockSize1": {
          "@id": "cov:blockSize1",
          "@type": "xsd:unsignedShort"
        }
      }
    }
  },
  "@type": "Form",
  "blockwise": {
    "@type": "BlockWiseTransferParameters",
    "blockSize1": 24
  }
}
```

```turtle
{
  "@context": {
    "cov": "https://www.w3.org/2019/wot/coap#",
    "hctl": "https://www.w3.org/2019/wot/hypermedia#",
    "xsd": "http://www.w3.org/2001/XMLSchema#",
    "Form": {
      "@id": "hctl:Form",
      "@context": {
        "blockwise": {
          "@id": "cov:blockwise",
          "@context": {
            "blockSize1": {
              "@id": "cov:blockSize1",
              "@type": "xsd:unsignedShort"
            }
          }
        }
      }
    }
  },
  "@type": "Form",
  "blockwise": {
    "blockSize1": 24
  }
}
```

</details>

Having `"@type": "@vocab"`` or not also doesn't seem to make a difference. For reference, I found [this in the JSON-LD spec](https://www.w3.org/TR/json-ld/#included-nodes), which I think is not what we're doing here.

So, I guess the best we can do is this (and people will need to apply `coap.ttl` after converting a TD to triples in order to infer the type of the `cov:blockwise` property value):

```suggestion
        "blockwise": {
            "@id": "cov:blockwise"
        },
        "qblockwise": {
            "@id": "cov:qblockwise"
        },
```